### PR TITLE
feat(config): add output default format

### PR DESCRIPTION
**Pull request by yfodil**

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -16,10 +18,36 @@ const (
...
+
+# Alias creates custom aliases for your Scaleway CLI commands
+{{ if .Alias }}alias:
+    aliases:
+        {{- range $alias, $commands := .Alias.Aliases }}
+        {{ $alias }}:
+        {{- range $index, $command := $commands }}
+            - {{ $command }}
+        {{- end }}
+       {{- end }}
```
Comment (quantumsheep) [inline internal/config/config.go:36]:
```suggestion
        {{- end }}
```

File: internal/config/config.go [hunk 2]
```diff
@@ -16,10 +18,36 @@ const (
...
+{{ else }}
+# alias:
+#     aliases:
+#         isl:
+#             - instance
+#             - server
+#             - list
+{{ end }}
```
Comment (quantumsheep) [inline internal/config/config.go:44]:
```suggestion
{{- end -}}
```

File: internal/core/bootstrap.go [hunk 3]
```diff
@@ -207,6 +203,18 @@ func Bootstrap(config *BootstrapConfig) (exitCode int, result interface{}, err e
 		return 1, nil, err
 	}
 	meta.CliConfig = cliCfg
+	if *cliCfg.Output != cliConfig.DefaultOutput && *cliCfg.Output != "" {
```
Comment (quantumsheep) [inline internal/core/bootstrap.go:206]:
```suggestion
	if *cliCfg.Output != cliConfig.DefaultOutput {
```

File: internal/config/config.go [hunk 4]
```diff
@@ -16,10 +18,36 @@ const (
...
+`
 )
 
 type Config struct {
-	Alias *alias.Config `json:"alias"`
+	Alias  *alias.Config `json:"alias"`
+	Output *string       `json:"output"`
```
Comment (quantumsheep) [inline internal/config/config.go:50]: Why is it a pointer if it's mandatory?

File: internal/core/bootstrap.go [hunk 5]
```diff
@@ -207,6 +203,18 @@ func Bootstrap(config *BootstrapConfig) (exitCode int, result interface{}, err e
...
+		outputFlag = *cliCfg.Output
+		printer, err = NewPrinter(&PrinterConfig{
+			OutputFlag: outputFlag,
+			Stdout:     config.Stdout,
+			Stderr:     config.Stderr,
+		})
+		if err != nil {
+			_, _ = fmt.Fprintln(config.Stderr, err)
```
Comment (quantumsheep) [inline internal/core/bootstrap.go:214]:
```suggestion
			fmt.Fprintln(config.Stderr, err)
```

I don't think it's required with `fmt.Fprintln`

**Review by quantumsheep — CHANGES_REQUESTED** [review]

**Review by quantumsheep — APPROVED** [review]

File: internal/config/config.go [hunk 6]
```diff
@@ -16,10 +18,37 @@ const (
...
+        {{- end }}
+{{- else }}
+# alias:
+#     aliases:
+#         isl:
+#             - instance
+#             - server
+#             - list
+{{- end -}}
+`
```
Comment (Codelax) [inline internal/config/config.go:46]: Make sure the created file has a newline at the end

**Review by Codelax — CHANGES_REQUESTED** [review]
Comment made outside the review

**Review by Codelax — APPROVED** [review]

**Review by Codelax — APPROVED** [review]

**Pull request merged**